casestudies/124/casestudy/www/layout/case_id_124_id_241_c_bio.html

```diff
@@ -214,7 +214,7 @@ <h1 class="title">
          <img src="../../files/photos/822/Tatiana.jpg" style="height: 272px; width: 250px;"/>
         </div>
         <p>
-         Tatiana Wah was director of the Haiti Research and Policy Program in the Center on Globalization and Sustainable Development at Columbia University�s Earth Institute. She advised the Haitian government on policy aimed at achieving the United Nations� Millennium Development Goals.
+         Tatiana Wah was director of the Haiti Research and Policy Program in the Center on Globalization and Sustainable Development at Columbia Universitys Earth Institute. She advised the Haitian government on policy aimed at achieving the United Nations Millennium Development Goals.
```

Comment:
It looks like some apostrophes became spaces

Reply:
Great catch - and I'm realizing what a nice job Github does in respecting file encodings. This is an excellent way of looking at the diffs.